contract/scripts/propose-start-contract.sh

```diff
@@ -20,3 +20,16 @@ agd tx gov submit-proposal swingset-core-eval "$PERMIT" "$SCRIPT" \
 set +x # not so noisy for this part
 . /usr/src/upgrade-test-scripts/env_setup.sh
 voteLatestProposalAndWait
```

Comment:
We're we already trying to approve on this side before?

Reply:
exactly.